content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/fefo.rst

```diff
@@ -0,0 +1,131 @@
+   <../../product_management/product_tracking/expiration_dates>` document.
+
+.. example::
+   In the :guilabel:`Inventory` tab of the product, `Egg`, the following :guilabel:`Dates` are set
+   by the user:
+
+   - :guilabel:`Expiration Date`: `30` days after receipt
+   - :guilabel:`Removal Date`: `15` days after expiration date
+
+   .. image:: fefo/user-set-date.png
+      :align: center
+      :alt: Display expiration and removal dates set on the product form.
+
+   Eggs, received from the vendor, arrive at the warehouse on January 1st. So, the expiration date
+   of the eggs is **January 31st** (Jan 1st + 30). By extension, the removal date is **January
+   16th** (Jan 31 - 15).
+
+.. _inventory/warehouses_storage/exp-date:
+
+To view the expiration dates of items in stock, navigate to the product form, and click the
+:guilabel:`On Hand` smart button.
+
+Next, click the additional options icon, located on the far-right, and select the columns:
+:guilabel:`Expiration Date` and :guilabel:`Removal Date`.
+
+.. image:: fefo/removal-date.png
+   :align: center
+   :alt: Show expiration dates from the inventory adjustments model accessed from the *On Hand*
+         smart button from the product form.
+
+Workflow
+--------
+
+Using the :abbr:`FEFO (First Expired, First Out)` removal strategy ensures that products with the
+nearest removal date are picked first.
+
+To understand how this removal strategy works, consider the following example below about the
+product, `Carton of eggs`, which is a box containing twelve eggs.
+
+The product is tracked :guilabel:`By Lots`, and the product category's :guilabel:`Force Removal
+Strategy` is set to :guilabel:`First Expired, First Out (FEFO)`.
+
+.. seealso::
+   - :ref:`Set up force removal strategy <inventory/warehouses_storage/removal-config>`
+   - :ref:`Enable lots tracking <inventory/warehouses_storage/lots-setup>`
+   - `Odoo Tutorials: Perishable Products <https://www.odoo.com/slides/slide/5324/share>`_
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * -
+     - LOT1
+     - LOT2
+     - LOT3
+   * - On-hand stock
+     - 5
+     - 2
+     - 1
+   * - Expiration date
+     - April 4
+     - April 10
+     - April 15
+   * - :ref:`Removal date <inventory/warehouses_storage/exp-date>`
+     - February 26
+     - March 4
+     - March 9
+
+To see the removal strategy in action, create a :ref:`delivery order <inventory/delivery/one-step>`
+for six Cartons of eggs, either by going to the :menuselection:`Sales app` and creating a new
+quotation, or from the delivery orders dashboard in :menuselection:`Inventory app --> Operations -->
+Deliveries`. :guilabel:`Confirm` the sales order, or click :guilabel:`Mark as Todo` on the draft
+delivery order.
+
+Doing so creates the delivery order for today, December 29th, and the lot numbers with the soonest
+expiration dates are reserved, thanks to the :abbr:`FEFO (First Expired, First Out)` removal
+strategy.
```

Comment:
Rewrite this to remove the workflow for creating a delivery order directly. I'll leave this comment on each of the docs as a reminder.